core/spring-authorization-server-core.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'io.spring.convention.spring-module'
+apply plugin: 'jacoco'
```

Comment:
Is this needed?

Reply:
You are right. Looks like applying the plugin was not needed. Removed this. 
Note that specifying the jacoco tool version is still needed for the build to pass on JDK 14.